homeassistant/components/home_connect/api.py

```diff
@@ -438,20 +438,36 @@ class CoffeeMaker(DeviceWithOpState, DeviceWithPrograms, DeviceWithRemoteStart):
     """Coffee maker class."""
 
     PROGRAMS = [
+        {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.Ristretto"},
         {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.Espresso"},
-        {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.EspressoMacchiato"},
+        {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.EspressoDoppio"},
         {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.Coffee"},
+        {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.XLCoffee"},
+        {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.EspressoMacchiato"},
         {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.Cappuccino"},
         {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.LatteMacchiato"},
         {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.CaffeLatte"},
-        {"name": "ConsumerProducts.CoffeeMaker.Program.CoffeeWorld.Americano"},
-        {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.EspressoDoppio"},
-        {"name": "ConsumerProducts.CoffeeMaker.Program.CoffeeWorld.FlatWhite"},
-        {"name": "ConsumerProducts.CoffeeMaker.Program.CoffeeWorld.Galao"},
         {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.MilkFroth"},
         {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.WarmMilk"},
-        {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.Ristretto"},
+        {"name": "ConsumerProducts.CoffeeMaker.Program.CoffeeWorld.KleinerBrauner"},
+        {"name": "ConsumerProducts.CoffeeMaker.Program.CoffeeWorld.GrosserBrauner"},
+        {"name": "ConsumerProducts.CoffeeMaker.Program.CoffeeWorld.Verlaengerter"},
+        {"name": "ConsumerProducts.CoffeeMaker.Program.CoffeeWorld.VerlaengerterBraun"},
+        {"name": "ConsumerProducts.CoffeeMaker.Program.CoffeeWorld.WienerMelange"},
+        {"name": "ConsumerProducts.CoffeeMaker.Program.CoffeeWorld.FlatWhite"},
```

Comment:
Not all of these programs are available on all models. Do we check the availability?
If not, I could understand this list might have been limited for a reason.

Reply:
For my understanding we don't check the availability. But Home Connect is only available to "high-class" coffee machines, so most of the machines will support all programs.

Reply:
Right. Maybe @DavidMStraub can shed light on this one?